core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status=None, job_id=None):
```

Comment:
Done. But @seanlip , we need the default args on job_id to decide if its getting updated or created. Someone creating a training job will not have access to job_id. Here, we could just reconsider making this function create_classifier_training_job instead. But, why is the way we currently do it wrong?